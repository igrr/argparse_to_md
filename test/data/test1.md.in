Some text before

<!--argparse_to_md:test1:get_parser-->

<!--argparse_to_md_end-->

some text after

<!--another comment-->
