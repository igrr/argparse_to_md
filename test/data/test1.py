import argparse

import module_which_does_not_exist


def get_parser() -> argparse.ArgumentParser:
    parser = argparse.ArgumentParser(prog='testprog', description='Description of the program')
    parser.add_argument('--foo', help='foo help')
    parser.add_argument('--bar', help='bar help')
    return parser
