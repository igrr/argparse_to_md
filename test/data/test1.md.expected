Some text before

<!--argparse_to_md:test1:get_parser-->
Usage:
```
testprog [-h] [--foo FOO] [--bar BAR]
```
Description of the program

Optional arguments:
- `--foo FOO`: foo help
- `--bar BAR`: bar help
<!--argparse_to_md_end-->

some text after

<!--another comment-->
